docs/routers/nokia-G240WB.md

```diff
@@ -0,0 +1,94 @@
+This guide was developed using a `Nokia G-240W-B` router (software version 3FE47969AGCA21). It should work for other models in the `G` series.
+
+!!! note
+    There is no single way to do it right. Chose the one best suited for you.
+This guide is `IPv4` and `IPv6` compatible, but it is not mandatory to use `IPv6`.
+
+### Prerequisites
+
+- `SSH` into your `Raspberry Pi` and get the `IPv4` and `IPv6` address with `ifconfig eth0` (substituting `eth0` for the applicable network interface).
+- The `Raspberry Pi` is using static IP address, `IPv4` and `IPv6` as well.
```

Comment:
```suggestion
- The `Raspberry Pi` is using static IP address, `IPv4` and `IPv6` as well. This can be a fixed DHCP assignment or configured statically on the Pi itself.
```